Corona-Warn-App/src/main/java/de/rki/coronawarnapp/risk/RiskLevel.kt

```diff
@@ -58,7 +56,6 @@ enum class RiskLevel(val raw: Int) {
             )
         private val HIGH_RISK_LEVELS = arrayOf(INCREASED_RISK)
         private val LOW_RISK_LEVELS = arrayOf(
-            UNKNOWN_RISK_INITIAL,
             NO_CALCULATION_POSSIBLE_TRACING_OFF,
             LOW_LEVEL_RISK,
             UNKNOWN_RISK_OUTDATED_RESULTS,
```

Comment:
What do we need this for?

Reply:
We were using it for displaying the grey risk cards. Basically before the risk calculation could take place after the onboarding, the initial risk score was attributed this constant value. By removing the grey card, I don't see a need for this constant anymore as the initial risk value is attributed to the LOW_RISK constant now.